src/Cache/Engine/FileEngine.php

```diff
@@ -413,8 +411,16 @@ protected function _setKey($key, $createKey = false)
      *
      * @return bool
      */
-    protected function _active()
+    protected function _dirWritable()
```

Comment:
There's always a possibility that something extends this class and uses that method.